src/components/popular-categories/PopularCategories.tsx

```diff
@@ -41,16 +41,17 @@ const PopularCategories: FC<PopularCategoriesProps> = ({
     () => categoryService.getCategories({ limit: itemsToShow }),
     [itemsToShow]
   )
-  const { response, loading } = useAxios<CategoryInterface[]>({
+  const { response, loading } = useAxios<ItemsWithCount<CategoryInterface>>({
     service: getCategories,
-    defaultResponse: defaultResponses.array
+    defaultResponse: defaultResponses.itemsWithCount
   })
 
+  const { items } = response
```

Comment:
is it necessary destructing here?

Reply:
no, it is not necessary, but we do this everywhere, so I decided to restructure here to

Reply:
fixed